### PR TITLE
Week2 hiren

### PR DESCRIPTION
**Pull request by hirenkumar91**
Dear Mentor all task done for week 2

### CODE REVIEW

File: app/about_us/aboutCSS/crew.module.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    font-size: 1.2rem;
+    text-align: justify;
+    word-spacing: 0.1rem;
+    padding: 3rem 0 2rem 0;
+  }
+  
+  .cardStack {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 3rem;}
```
Comment (vdaguenet) [inline app/about_us/aboutCSS/crew.module.css]: Mind the indentation and code formatting here

File: app/about_us/aboutCSS/partners.module.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.heading {
+    padding-top: 3rem;
+    font-size: 1.5rem;
+    text-align: center;
+    margin-bottom: 2rem;
+    color: white;
+  }
+  
+  .logoContainer {
```
Comment (vdaguenet) [inline app/about_us/aboutCSS/partners.module.css]: Indentation

File: app/about_us/ourCrew.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        imageUrl: "./crew/image-anousheh-ansari.png",
+        description:
+          "Maya brings a unique blend of technical skills and customer service experience to the team. She’s always ready to assist with any needs and to make sure every traveler has an unforgettable experience.",
+      },
+];
+
+const CrewDetails =({ name, title,imageUrl,description}) =>{
+    return(
+    <div className={styles.crewCard}>
+    <div class={styles.cardImg}>
```
Comment (vdaguenet) [inline app/about_us/ourCrew.js]: The JSX needs to be indented like normal HTML

File: components/ui/Footer.js [hunk 4]
```diff
@@ -1,53 +1,73 @@
...
-
-      {/* TASK - React 1 week 1 */}
-      {/* Add a new list item for LINKEDIN */}
       <div className={styles.footerLinks}>
-        <h3>Follow us</h3>
+        <h3>Pages</h3>
         <ul className={styles.footerList}>
           <li>
-            <a href="https://facebook.com">Facebook</a>
+            {" "}
```
Comment (vdaguenet) [inline components/ui/Footer.js]:
```suggestion
            
```
You don't need this extra empty space here

**Review by vdaguenet — APPROVED** [review]
Be careful with indentation. The same rules that you learned so far also apply to React.

File: components/ui/Footer.js (hunk 4 above)
Comment (hirenkumar91) [reply components/ui/Footer.js]: done

File: app/about_us/aboutCSS/crew.module.css (hunk 1 above)
Comment (hirenkumar91) [reply app/about_us/aboutCSS/crew.module.css]: indentation done

File: app/about_us/aboutCSS/partners.module.css (hunk 2 above)
Comment (hirenkumar91) [reply app/about_us/aboutCSS/partners.module.css]: indentation done

File: app/about_us/ourCrew.js (hunk 3 above)
Comment (hirenkumar91) [reply app/about_us/ourCrew.js]: indentation done

**Comment by hirenkumar91** [comment]
Dear Mentor all updates done as suggested. should i meagre branch

**Review by vdaguenet — APPROVED** [review]

**Pull request merged**